### PR TITLE
Don't silence Hugo

### PR DESCRIPTION
**Pull request by thattommyhall**
This was masking errors when running from Make

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -64,4 +64,4 @@
     "webpack": "^4.41.0",
     "webpack-cli": "^3.3.9"
   }
-}
+}
```
Comment (aschmahmann) [inline package.json:67]: I think these files seem to like newlines at the end for some reason

File: package.json [hunk 2]
```diff
@@ -12,7 +12,7 @@
     "webpack:watch": "webpack --watch-stdin",
     "less:watch": "onchange 'site/assets/less/**/*.less' -- npm run less:build",
     "less:build": "lessc site/assets/less/main.less --clean-css site/static/site.css",
-    "build:hugo": "hugo --quiet -s site -d ../releases",
```
Comment (aschmahmann) [inline package.json:15]: Seems reasonable to me, although I know almost nothing about Hugo so I'll defer to @lidel to double check me.

**Comment by hugomrdias** [comment]
Thank you 😆

**Comment by olizilla** [comment]
I silenced hugo cos he talks to much. but masking errors is unhelpful, so go for it.

**Pull request merged**